inst/startup/worker.R

```diff
@@ -71,7 +72,24 @@ setwd(batchTaskWorkingDirectory)
 
 azbatchenv <-
   readRDS(paste0(batchJobPreparationDirectory, "/", batchJobEnvironment))
-argsList <- azbatchenv$argsList
+
+tryCatch({
+  print("Data Set Boolean: ")
```

Comment:
Kind of a hard to decipher print output. Can you make this (and lines 79 & 80) a bit more verbose?